File: Ruby_/entradaSaida/Lista4.rb
# frozen_string_literal: true

print "Digite quantos anos você tem: "
anos = gets.to_i
print "Digite quantos meses faltam para seu próximo aniversário: "
meses = gets.to_i
print "Digite quantos dias faltam para o dia do próximo mês com o mesmo dia do seu aniversário: "
dias = gets.to_i

anos *= 365
meses *= 30

puts "Desde que você nasceu se passaram: #{anos + meses + dias}"

File: Kotlin/entradaSaida/Lista4.kt
package entradaSaida

fun main(){
    // entrada de dados
    print("Digite quantos anos você tem: ")
    var anos = readln().toInt()
    print("Digite quantos meses faltam para seu próximo aniversário: ")
    var meses = readln().toInt()
    print("Digite quantos dias faltam para o dia do próximo mês com a mesmo dia do seu aniversário: ")
    val dias = readln().toInt()

    // atribuindo valores
    anos *= 365
    meses *= 30

    println("Desde que você nasceu se passaram: ${dias + meses + anos} dias")
}